Attempting to create an alias named ".." results in panic.

talias crashes when you attempt to create an alias named .. as it should since . and .. may want to do some light validation of alias names.

```bash
steve@~$ talias -a ..
9  |  talias -l
10  |  talias -L
11  |  ls -al
12  |  cd ..
13  |  cat .viminfo
14  |  pwd
15  |  talias -d
16  |  exit
17  |  cd ~
18  |  talias -l
Enter command number (or 'p' for previous 10) : 12
panic: open /home/steve/.talias: is a directory

goroutine 1 [running]:
main.check(0x58b420, 0xc42000e690)
        /home/steve/talias/talias.go:108 +0x4a
main.addAliasScript(0xc420014350, 0x6, 0xc, 0x0, 0x7fffdf670a79, 0x2, 0x410b00)
        /home/steve/talias/talias.go:239 +0xfd
main.addAlias(0xc42000e660, 0xc42000e630)
        /home/steve/talias/talias.go:270 +0x1d6
main.main()
        /home/steve/talias/talias.go:343 +0x15a
```

--- a/talias.go
+++ b/talias.go
@@ -3,6 +3,7 @@ package main
 import (
 	"bufio"
 	"fmt"
+	"errors"
 	"os"
 	"strconv"
 	"flag"
@@ -336,6 +337,10 @@ func addAliasScript(info CmdInfo, alias string) bool {
 
 // Adds an alias to the database and creates its script
 func addAlias(cmdMap ShellCmdMap, taliasData TaliasCmdMap) {
+	if ctx.addAliasName == ".." || ctx.addAliasName == "." {
+		check(errors.New("Error: Alias can not be named '..' or '.'."))
+	}
+
 	existingTalias := taliasData[ctx.addAliasName]
 	if existingTalias.Alias != "" {
 		fmt.Print(ctx.addAliasName, " exists. Extend for ", ctx.expiration.Hours(),

